src/global.d.ts

```diff
@@ -35,6 +35,7 @@ declare module 'clientConfig' {
     coreName?: string;
     solrQueryConfig?: SolrQueryConfig;
     activateLayerOnClick?: boolean;
+    searchResultDrawer?: boolean;
```

Comment:
I would recommend to rename the config to a more meaningful name expecting a boolean value like `useSearchResultDrawer`, `showSearchResultDrawer` or similiar.

Reply:
I have renamed this config to `showSearchResultDrawer` :+1: